docker-compose-releng.yml

```diff
@@ -0,0 +1,90 @@
+version: "2.1"
+
+# TODO: what links do we need?
+services:
+  releng_db:
+    image: postgres@sha256:6ac0fbeddde3bb7b0003a2ace8c126f742f8bdd90695801337d3edaaf1fcc478
+    expose:
+      - 5432
+    environment:
+      - POSTGRES_PASSWORD=releng
+      - POSTGRES_USER=releng
+      - POSTGRES_DB=releng
+    volumes:
+      - ./tmp/postgres:/var/lib/postgresql/data
+
+  releng_frontend:
+    image: mozbhearsum/bhearsum-shipit-test
+    ports:
+      - "8000:8000"
+    volumes:
+      - .:/app
+    working_dir: /app
+    environment:
+      - APP_DEV_HOST=0.0.0.0
+      - SERVICES_APP=releng_frontend
+
+  releng_clobberer:
+    image: mozbhearsum/bhearsum-shipit-test
+    ports:
+      - "8001:8001"
+    volumes:
+      - .:/app
+    working_dir: /app
+    environment:
+      - APP_DEV_HOST=0.0.0.0
+      - SERVICES_APP=releng_clobberer
+
+  releng_tooltool:
+    image: mozbhearsum/bhearsum-shipit-test
+    ports:
+      - "8002:8002"
+    volumes:
+      - .:/app
+    working_dir: /app
+    environment:
+      - APP_DEV_HOST=0.0.0.0
+      - SERVICES_APP=releng_tooltool
+
+  releng_treestatus:
+    image: mozbhearsum/bhearsum-shipit-test
+    ports:
+      - "8003:8003"
+    volumes:
+      - .:/app
+    working_dir: /app
+    environment:
+      - APP_DEV_HOST=0.0.0.0
+      - SERVICES_APP=releng_treestatus
+      - DATABASE_URL=postgresql://releng:releng@releng_db:5432/releng
+    links:
+      - releng_db
+    healthcheck:
+      # TODO: this command generates spamming output from the releng_db container
+      # using the real pg client might fix this
+      test: ["CMD", "nc", "-vz", "releng_db", "5432"]
```

Comment:
While these work, it works out I did them wrong. Healthchecks should be part of the container they're checking (so this one should be in the db container), and there's a special depends_on syntax for depending on a service being healthy. https://github.com/mozilla/balrog/pull/297#pullrequestreview-31566088 has a good example of how to do it.